### ISSUE TITLE
Remove/make optional JMSDiExtraBundle dependency

### ISSUE THREAD

**Issue by okwinza** [tarioch/pheal-bundle#1]
It's more like a friendly advice, but IMO if this bundle was meant to be reusable then this dependency is quite redundant.

**Comment by tarioch** [comment]
Maybe I'm missing something but isn't this required to be able to use @DI\Service and similar annotations?
I know that instead of the annotations I can also use the yaml/xml configuration but I prefer the annotation way.

**Comment by okwinza** [comment]
Yep, JMSDiExtraBundle is required for describing DI via annotations and that's the thing since it's not a mainstream approach(for 3rd party libs at least).

Having being forced to install additional bundle just to make this one work may be quite limiting for someone.

For instance, this is what made me write my own PhealNGBundle for my eve-related project not so long ago.
https://github.com/EvELabs/EvelabsPhealNGBundle
It's pretty much identical to yours but uses traditional .yaml for configs.

Or we could also make this dependency optional like this:
https://github.com/schmittjoh/JMSSerializerBundle/pull/313/files

Thanks for blazing fast reply by the way :+1:

**Comment by tarioch** [comment]
In that case I might actually simply drop this and move to use your implementation. Thanks for letting me know.

**Comment by okwinza** [comment]
:+1: That would be awesome! I will update the readme (& maybe add some new stuff too) tomorrow.
